graylog2-web-interface/src/components/navigation/Navigation.tsx

```diff
@@ -88,6 +88,9 @@ const Navigation = React.memo(({ pathname }: Props) => {
   const enterpriseMenuIsMissing = !pluginExports.find((value) => value.description.toLowerCase() === 'enterprise');
   const isPermittedToEnterprise = isPermitted(permissions, ['licenseinfos:read']);
 
+  const securityMenuIsMissing = !pluginExports.find((value) => value.description.toLowerCase() === 'analyst tools');
```

Comment:
What do you think about creating a constant which can be reused in the enterprise repo, to ensure we do not change on part without the other? We could do the same for the enterprise menu bindings.

Reply:
Abstracted into the `Routes.ts` file.